webui/src/scss/_button-edit.scss

```
@@ -173,21 +173,58 @@
 		display: flex;
 		flex-direction: row;
 		margin-right: -0.75rem;
+		min-width: 0;
 
 		.cell-name {
 			flex-grow: 1;
+			min-width: 0;
 
 			font-weight: bold;
```

Stylelint (17.14.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)